### ISSUE TITLE
Schedule or callLater Request()

### ISSUE THREAD

**Issue by franciscolourenco** [scrapy/scrapy#169]
Is there a way to schedule a Request for a specific url for later, instead of having the yield Request() be processed asap?

**Comment by osya** [comment]
I tried to use it with Scrapy 0.24.4 and the following error occures "Spider must return Request, BaseItem or None, got 'instance' in ". Please advise

**Comment by tarlabs** [comment]
@osya, This works fine for me in the latest version

@pablohoffman, Is it possible to make something on this as a part of Scrapy?

The use case is like this, we POST/Search through a form. That form generates a link. But it actually redirects you to that link after 15 seconds, when the results are generated. So the next url can be know but needs to be called only after 15 sec. Unfortunately I was using a sleep for this, which made scraper go to around 4 hours and now with the deferred approach it is just taking 22 mins. Would be nice to have some special meta or DelayedRequest class for this

**Comment by Curita** [comment]
@tarlabs: this issue is being discussed in #802, feel free to join in.
